fixtures/CAPI.ts

```diff
@@ -400,4 +400,6 @@ export const CAPI: CAPIType = {
         editionAdTargeting: [],
     },
     starRating: 2,
+    trailText:
+        'Ticket touts face unlimited fines for using ‘bots’ to buy in bulk',
```

Comment:
It'd be nice to be able to test against a selection of inputs / chars, but I guess we trust trailText because we're `JS Object` -> `JS Object`? Just wondering about malformed markup in some way.

Reply:
Yeah, it's a good point. I don't think trailtext can contain HTML but will check!